cmd/ethrex_l2/src/commands/test.rs

```diff
@@ -154,6 +195,161 @@ async fn test_connection(cfg: EthrexL2Config) -> bool {
     false
 }
 
+async fn wait_receipt(
```

Comment:
We already have a definition for this function [here](https://github.com/lambdaclass/ethrex/blob/main/cmd/ethrex_l2/src/commands/wallet.rs#L599), though it does not do retries. We could use your implementation as the new function and have it on `utils`.